geode-core/src/main/java/org/apache/geode/distributed/internal/InternalDistributedSystem.java

```diff
@@ -3091,4 +3068,22 @@ public void stopReconnecting() {
 
     Throwable generateCreationStack(final DistributionConfig config);
   }
+
+  public void requestRegisterBean(Function function) {
```

Comment:
DistributedSystem should limit its concerns to membership and clustering. These are new methods for MBeans and Functions -- InternalDistributedSystem should not care about MBeans and Functions. ManagementService already has some User APIs for MBeans.